ts_src/psetv2/updater.ts

```diff
@@ -46,6 +46,30 @@ export interface AddInReissuanceArgs {
   tokenAssetBlinder: string | Buffer;
 }
 
+export interface UpdaterInput {
+  txid: string;
+  txIndex: number;
+  sequence?: number;
+  heightLocktime?: number;
+  timeLocktime?: number;
+  witnessUtxo?: TxOutput;
+  nonWitnessUtxo?: Transaction;
+  sighashType?: number;
+  tapInternalKey?: TapInternalKey;
+  tapLeafScript?: TapLeafScript;
+  tapMerkleRoot?: TapMerkleRoot;
+  issaunceOpts?: IssuanceOpts;
+  reissuanceOpts?: ReissuanceOpts;
+}
+
+export interface UpdaterOutput {
+  asset: string;
+  amount: number;
```

Comment:
shall amount be optional for example for dummy conf outputs?
```suggestion
  amount?: number;
```

Reply:
I kept the same requirement as `CreatorOutput`. I think the user should specify explicitly the amount.